src/commands/genericcmds.cpp

```diff
@@ -55,6 +56,7 @@ namespace
 			value("sqf-vm"),
 			value(VERSION_MAJORMINOR),
 			value(VERSION_REVISION),
+			value(g_GIT_SHA1),
```

Comment:
The SHA1 needs to either be joined into one of the existing variants or replace an existing.
See https://community.bistudio.com/wiki/productVersion

Reply:
Oh I was wondering why the command didn't have __ at the end! Perhaps putting it in this command is wrong then.

Reply:
If it was to replace something I guess branch would be the right thing maybe?

Reply:
I'd do majorminor.sha1
`value(VERSION_MAJORMINOR"."g_GIT_SHA1)`

Reply:
branch would be my prefered variant too
something like `community (git sha1 xxxxxxx)` maybe?